tests/ui/specs/FanSpeedControl/FanSpeedControl-specs.js

```diff
@@ -0,0 +1,69 @@
+const Page = require('./FanSpeedControlPage');
+
+describe('FanSpeedControl', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('default', function () {
+		const fanSpeedControl = Page.components.fanSpeedControlDefault;
+
+		it('should have the first arc selected by default', function () {
+			expect(fanSpeedControl.coloredPath(1).getCSSProperty('stroke').value).to.equal('rgb(68,68,68)');
```

Comment:
In fact, there is a screenshot test for this part, so a function-oriented test should be conducted in the ui test.
Anyway, it's an implemented part. So we just keep it.